app/main.ipynb

```diff
@@ -126,19 +170,28 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
-    "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1])\n",
-    "# write your code here"
+    "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1], dtype=np.float32)\n",
+    "result = array[-2] + array[0] + array[3]\n",
+    "print(np.int8(result))"
```

Comment:
The calculation in Task 4 does not produce the expected result of 27. You should review the indices used for multi-indexing to ensure they sum up to 27 as required by the task description.